### PR TITLE
Improve documentation coverage for Bertweet

### LINKED ISSUE #9035: Improve coverage of the documentation
Currently, some public classes are not documented anywhere because we didn't create the corresponding doc pages. Those missing pages are:

- Benchmark classes
- Bert Japanese
- Data collators

If someone feels like working on one of those, please tag yourself with a comment on this issue. Once the objects are properly documented, they can be removed from the `SHOULD_BE_DOCUMENTED` constant in [this file](https://github.com/huggingface/transformers/blob/1310e1a758edc8e89ec363db76863c771fbeb1de/utils/check_repo.py#L374).

### PR DESCRIPTION
**Pull request by Qbiwan**
# What does this PR do?

Fixes #9035
@sgugger  added docs for Bertweet

## Before submitting

- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors which may be interested in your PR.

### CODE REVIEW

File: docs/source/model_doc/bertweet.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+The abstract from the paper is the following:
+
+*We present BERTweet, the first public large-scale pre-trained language model for English Tweets. Our BERTweet, having
+the same architecture as BERT-base (Devlin et al., 2019), is trained using the RoBERTa pre-training procedure (Liu et
+al., 2019). Experiments show that BERTweet outperforms strong baselines RoBERTa-base and XLM-R-base (Conneau et al.,
+2020), producing better performance results than the previous state-of-the-art models on three Tweet NLP tasks:
+Part-of-speech tagging, Named-entity recognition and text classification.*
+
+.. code-block::
```
Comment (sgugger) [inline docs/source/model_doc/bertweet.rst:30]: Let's add a small introduction here, like  "Example of use:"

**Review by sgugger — APPROVED** [review]
Great, thanks for contributing this! I left one suggestion and I think `BertweetTokenizer` should then be removed from the variable `SHOULD_HAVE_THEIR_OWN_PAGE` of the `utils/check_repo.py` file.

File: docs/source/model_doc/bertweet.rst (hunk 1 above)
Comment (Qbiwan) [reply docs/source/model_doc/bertweet.rst:32]: thanks for the review. I removed `BertweetTokenizer` from **utils/check_repo.py** and added `Example of use:` to **bertweet.rst**.

**Review by LysandreJik — APPROVED** [review]
LGTM!

**Comment by LysandreJik** [comment]
Thanks @Qbiwan!

**Pull request merged**